src/CreateMiddleware/CreateMiddleware.php

```diff
@@ -41,10 +41,11 @@ public function process(ServerRequestInterface $request, DelegateInterface $dele
     /**
      * @param string $class
      * @param string|null $projectRoot
+     * @param string|null $classSkeleton
      * @return string
      * @throws CreateMiddlewareException
      */
-    public function process($class, $projectRoot = null)
+    public function process($class, $projectRoot = null, $classSkeleton = null)
```

Comment:
Depends of PHP version we have got warning or strict standards error. 

https://3v4l.org/ONv3N

```php
class A {
    public function x($p = null)
    {
    }
}

class B extends A {
    public function x() {
        return __CLASS__;
    }
}

$b = new B;
var_dump($b->x());
```

```
Output for 7.0.0 - 7.2.0alpha2
Warning: Declaration of B::x() should be compatible with A::x($p = NULL) in /in/ONv3N on line 15
string(1) "B"
```

```
Output for 5.6.0 - 5.6.30
Strict Standards: Declaration of B::x() should be compatible with A::x($p = NULL) in /in/ONv3N on line 15
string(1) "B"
```